frontend/app/component/error/Error.module.css

```
@@ -0,0 +1,38 @@
+.errorContainer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.errorContainer h1 {
+  font-size: var(--font-size-xl);
+  font-family: var(--font-text);
+  font-weight: 700;
```

```suggestion
  font-weight: bold;
```
we always use `bold` instead of 700 and `normal` instead of 400 to make it more readable :)

---

Fixed in 53ffbfba4f703f64c491a1eb853a092e4c62b807